redaxo/src/core/backend.php

```diff
@@ -7,6 +7,31 @@
 header('X-Robots-Tag: noindex, nofollow, noarchive');
 header('X-Frame-Options: SAMEORIGIN');
 
+// assets which are passed with a cachebuster will be cached very long,
+// as we assume their url will change when the underlying content changes
+if (rex_get('asset') && rex_get('buster')) {
+    $assetFile = rex_get('asset');
```

Comment:
anstatt den font-awesome font anzupassen etc. würde ich hier lieber kurz vor der ausgabe alle relativen pfade in der ausgabe "umschreiben"